package.json

```diff
@@ -59,12 +59,14 @@
     "eslint-config-prettier": "^2.6.0",
     "font-awesome": "^4.7.0",
     "linkifyjs": "^2.1.6",
+    "moment": "^2.21.0",
```

Comment:
Make sure we're not suffering from this: https://github.com/jmblog/how-to-optimize-momentjs-with-webpack